File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program86.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is alphabet or not\ (A-Z a-z).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Solution
{
	private:
		char ch;
		
	public:
		Solution(char ch='\0')
		{
			this->ch = ch;
		}

		bool ChkAlpha()
		{
			if(ch>='A' && ch<='Z' || ch>='a' && ch<='z')
			{
				return true;
			}
			else
			{
				return false;
			}


		}


};

int main()
{
	char ch='\0';
	bool bRet=false;

	cout<<"Enter the character: ";
	cin>>ch;

	Solution *dObj = new Solution(ch);
	bRet = dObj->ChkAlpha();

	if(bRet==true)
	{
		cout<<"It is alphabet\n";
	}
	else
	{
		cout<<"It is not alphabet\n";
	}

	delete dObj;
	return 0;
}

/*
OUTPUT:
Enter the character: F
It is alphabet

*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program86.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is alphabet or not\ (A-Z a-z).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{

	public char ch;

	public Solution(char ch)
	{
		this.ch = ch;
	}

	public boolean ChkAlpha()
	{	
		
		if(ch>='a' && ch<='z' || ch>='A' && ch<='Z')
		{
			return true;
		}
		else
		{
			return false;
		}
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);
		boolean bRet = false;

		System.out.println("Enter character:");
		char ch = sObj.next().charAt(0);

		Solution obj = new Solution(ch);
		bRet = obj.ChkAlpha();

		if(bRet == true)
		{
			System.out.println("It is Alphabet");
		}
		else
		{
			System.out.println("It is not Alphabet");
		}
	}
}

/*
OUTPUT:
Enter character:
A
It is Alphabet


*/
